README.md

```diff
@@ -12,13 +12,52 @@ The Kiva UI project is bound by a [Code of Conduct](code_of_conduct.md).
 
 Kiva welcomes outside contributions to our UI repository. If you have any ideas for a feature or improvement, create an issue and we can discuss whether it makes sense to create a pull request. Thanks for the help!
 
-## Build Setup for localhost (outside of a Kiva VM or Server)
+# Local Development Setup with Caddy
+
+> IMPORTANT NOTE: Turn off Docker if it's running! We have a perpetually running network related to Tilt that will prevent Caddy from starting. I did consider using Tilt and docker-compose but it would require rewriting alot of the Monolith TiltVM setup so this just bypasses it all.
+
+### Required Dependencies
+
+1. Add `127.0.0.1 kiva-ui.local` to your `/etc/hosts` file on your mac
+	- Auth0 configs are already in place to support this domain in the dev tenant
+
+2. Install Caddy
+
+	 `brew install caddy`
+
+3. Start Caddy from the root of the ui repo
+
+	`caddy start`
```

Comment:
Doesn't need to change, but I use `caddy run` so I can watch for failures (not usually important though...).